src/mobu/business/notebookrunner.py

```diff
@@ -113,10 +113,11 @@ async def lab_business(self) -> None:
 
             for cell in cells:
                 self.running_code = "".join(cell["source"])
-                await self.execute_code(kernel, self.running_code)
+                await self.execute_code(session, self.running_code)
 
         self.running_code = None
-        await self.delete_kernel(kernel)
+        await self.idle()
```

Comment:
Not sure why you're adding an idle here. It feels like we should be able to close the session immediately after executing the code?

Reply:
I think I was just keeping the same structure as our other loops?  We don't need it, and I'll remove it.